routes/application/index.coffee

```diff
@@ -102,6 +110,7 @@ class Application extends React.Component
 
       <h1>is a vote for</h1>
       <img src={@bestOption().img} alt={"The #{@bestOption().name}"}/>
+      <h3>{@bestOption().name}</h3>
```

Comment:
☠️ I think the image is enough

Reply:
I suppose, I just figured it might be better for some accessibility / reading software, but the alt text probably covers that.